pilot/pkg/networking/core/v1alpha3/listener.go

```diff
@@ -290,6 +303,9 @@ func (configgen *ConfigGeneratorImpl) buildSidecarOutboundListeners(env model.En
 			listenAddress := WildcardAddress
 			var addresses []string
 			var listenerMapKey string
+			var networkFilters []listener.Filter
```

Comment:
Here and elsewhere, try to avoid pre-declaring variables. E.g. for the filters, we only need them once to create the params; just declare them there.

Reply:
see above